safe/report/processors/default.py

```diff
@@ -725,7 +725,7 @@ def atlas_renderer(composition, coverage_layer, output_path, file_format):
         has_project_scales = QgsProject.instance().readBoolEntry(
             "Scales", "/useProjectScales")[0]
         if not has_project_scales or not scales:
-            scales_string = str(general_setting("Map/scales", PROJECT_SCALES))
+            scales_string = str(setting("Map/scales", PROJECT_SCALES))
```

Comment:
So it switches automatically to QGIS settings or inasafe settings? (what about the prefix 'inasafe/'?)